ui/v2.5/src/components/List/Pagination.tsx

```diff
@@ -96,6 +101,12 @@ export const Pagination: React.FC<IPaginationProps> = ({
         <span className="d-none d-sm-inline">Last</span>
         <span className="d-inline d-sm-none">&#x300b;</span>
       </Button>
+      <Button
+        variant="secondary"
+        disabled={true}
+      >
+        {indexText}
+      </Button>
```

Comment:
Not entirely sure how I feel about a disabled button. I honestly think it looks fine with just a `<div>`, it strongly implies that it's not part of the actual navigation.

Reply:
Eh, I disagree, but more fundamentally -- I think an index is part of the navigation of a paginated structure. Yes, it's not a functional part, but it conveys key information about navigation to the user.